<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SDN-271" name="ICIX Test For Krishna Org" tests="1" failures="1" timestamp="24 Feb 2017 09:16:12 GMT" time="154.897" errors="0">
  <testcase name="SearchByCompanyName" time="106.929" classname="icix.Tests.TC9648_Test">
    <failure type="java.lang.AssertionError" message="expected [We found 1 match in the ICIX Network.] but found [We found No matches in the ICIX Network.]">
      <![CDATA[java.lang.AssertionError: expected [We found 1 match in the ICIX Network.] but found [We found No matches in the ICIX Network.]
	at icix.Utils.Common.assertText(Common.java:493)
	at icix.Modules.Account.searchTPBy(Account.java:46)
	at icix.Tests.TC9648_Test.SearchByCompanyName(TC9648_Test.java:31)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- SearchByCompanyName -->
</testsuite> <!-- ICIX Test For Krishna Org -->
